### PR TITLE
docker-machine v0.5.1

### PR DESCRIPTION
**Pull request by gvangool**
Update docker-machine to v0.5.1.

This supersedes #46065

### CODE REVIEW

**Comment by kad** [comment]
Let's synchronize updates. v0.5.1 will break docker-machine-parallels, see Parallels/docker-machine-parallels#18 for details. I'll provide updated formula for docker-machine-parallels, or you can just update it to v1.1.0 release of docker-machine-parallels.

**Comment by gvangool** [comment]
@kad updated pull request with new docker-machine-parallels.

File: Library/Formula/docker-machine-parallels.rb [hunk 1]
```diff
@@ -3,8 +3,8 @@
 class DockerMachineParallels < Formula
   desc "Docker Machine Parallels Driver"
   homepage "https://github.com/Parallels/docker-machine-parallels"
-  url "https://github.com/Parallels/docker-machine-parallels/archive/v1.0.1.tar.gz"
-  sha256 "9a5abc454748aeb83c74c45efd3a73359a47e408e215318fbcc6446026581652"
+  url "https://github.com/Parallels/docker-machine-parallels/archive/v1.1.0.tar.gz"
+  sha256 "53f31c7d675c4ea7525466fd9a41cd80869754637bf42390ba9666d54b9ccc6e"
   head "https://github.com/Parallels/docker-machine-parallels.git"
```
Comment (kad) [inline Library/Formula/docker-machine-parallels.rb]: this checksum is for binary downloads.
for source it should be 0f2ccc1c470a71b40e81b548b4587cdcbc098f0ceeea6295ec8bebbb06aa9135

**Comment by kad** [comment]
Heh, in the end of the day, v1.1.0 release is not that easy. I've created new PR #46227
It passes my local checks, but let's see what CI thinks.

**Pull request closed without merging**